examples/connection_pools/migrations/2018-02-20-033155_create_sample_table/down.sql

```diff
@@ -0,0 +1,2 @@
+-- This file should undo anything in `up.sql`
```

Comment:
Are these migration files doing anything?

Reply:
No. They're not. In fact, do you think we even need the `connection_pools` example now that we have `todo` using the feature?

I never did anything more sophisticated than made Rocket happy to store the pool in state, which is more of an integration-level test than a true example.

I think we're better off deleting this example and just going with `todo`.

Reply:
The only this example appears to have that `todo` doesn't is that it demonstrates multiple database support.

Reply:
That's a fair point.